### ISSUE TITLE
Installation fails on Centos 6.6

### ISSUE THREAD

**Issue by marcolino** [h2non/youtube-video-api#20]
```
$ npm install youtube-video-api

> weak@1.0.0 install /var/www/html/youtube/server/node/node_modules/youtube-video-api/node_modules/nightmare/node_modules/phantom/node_modules/dnode/node_modules/weak
> node-gyp rebuild

make: Entering directory `/var/www/html/youtube/server/node/node_modules/youtube-video-api/node_modules/nightmare/node_modules/phantom/node_modules/dnode/node_modules/weak/build'
  CXX(target) Release/obj.target/weakref/src/weakref.o
In file included from /home/msolari/.node-gyp/4.0.0/include/node/node.h:42,
                 from ../node_modules/nan/nan.h:24,
                 from ../src/weakref.cc:18:
/home/msolari/.node-gyp/4.0.0/include/node/v8.h:336: error: expected unqualified-id before ‘using’
/home/msolari/.node-gyp/4.0.0/include/node/v8.h: In constructor ‘v8::MaybeLocal<T>::MaybeLocal()’:
...
...
...
```

_Any clue?_

**Comment by h2non** [comment]
There's an error compiling a native `node.js` module. Which version or `node.js` are you using? Try upgrading and reinstalling `node-gyp`.

**Comment by marcolino** [comment]
I solved upgrading g++ from 4.7 to 4.8.1.
Thanks.
